iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergOutputCommitter.java

```diff
@@ -413,6 +415,7 @@ private void commitTable(FileIO io, ExecutorService executor, OutputTable output
 
     Table table = null;
     String branchName = null;
+    boolean isCompaction = false;
```

Comment:
I'm bit problematic with the variable names. 
IsComapction in my mind does not express rightly the variable and the domain model.
I would suggest:

- isCompactionEnabled
- compactionEnabled
- comapctionStatus (Enum in this case maybe better)
- isCompactionActive
- allowCompaction 

I think those variable + getter setter would much more express the importance of this variable. cc: @ayushtkn @dengzhhu653 Please disagree if this is not the hive style.

Reply:
I'm ok with the name, looks this variable tends to use internally only in this class(no set/get).